specification/hugr.md

```diff
@@ -138,6 +144,15 @@ EdgeKind ::= Hierarchy | Value(Locality, SimpleType) | Static(Locality, ClassicT
 
 Locality ::= Local | Ext | Dom
 ```
+
+Ports always exist on `Value` and `Static` edges; never on `Hierarchy`, `Order `
+or `ControlFlow` edges. Every port has an associated `SimpleType`. The source
```

Comment:
What about the ordered side of `ControlFlow` edges? Don't we require ports there?

Reply:
We need an ordering there yes, but we may wish to use different language in the spec than "Ports". I'm in two minds tbh.

Reply:
In two minds as well. It is only the order that matters so this seems different from ports as carriers of specific types. But indeed this ordering is not otherwise captured...